website/docs/r/muting_schedule.html.markdown

```diff
@@ -120,4 +121,8 @@ The following arguments are supported:
   - `ids` - (Optional) List of monitor Ids in hex. Must be empty if `all` is true.
   - `all` - (Optional) True if the schedule applies to all monitors
 
+#### notification_group_type
```

Comment:
Please add an example that demonstrates usage of `notification_groups`.

Reply:
added.

Reply:
Since we don't have the concept of types in notification_group, `notification_group_type` sounds a little misleading? Can we get rid of the `_type` suffix?

Reply:
OK. let me remove the suffix.